.github/workflows/documentation.yml

```diff
@@ -28,5 +28,6 @@ jobs:
       - name: Build docs
         run: |
           cd docs
-          . ./setup-env.sh
+          ./setup-env.sh
+          . ../.venv/bin/activate
```

Comment:
Just out of curiosity, why was this line needed? Activate appears to get sourced in in the setup-env.sh script? I'm assuming it's some shell or sequencing thing ...